src/action/payment.js

```diff
@@ -252,6 +290,16 @@ class PaymentAction {
     }
   }
 
+  async _sendPayment() {
+    const { payment, settings } = this._store;
+    let amount = payment.sendAll ? 0 : toSatoshis(payment.amount, settings);
+    await this._grpc.sendCommand('sendCoins', {
+      addr: payment.address,
+      amount,
+      sendAll: payment.sendAll,
+    });
+  }
```

Comment:
Nice pulling out a helper function here. Makes the `payBitcoin` more readable too.